files/image_config/copp/copp_cfg.j2

```diff
@@ -97,7 +97,7 @@
 		    "trap_group": "queue4_group3"
 	    },
 	    "dhcp_relay": {
-		    "trap_ids": "dhcp,dhcpv6",
+		    "trap_ids": "dhcpv6",
```

Comment:
DHCP trap is required to redirect and police the DHCP traffic in ASIC before hitting the kernel/control plane. Kernel level DHCP rate-limiting in https://github.com/sonic-net/sonic-swss/pull/3130 may be required for software platforms, but this trap is required for all hardware switches.